source/functions.rst

```diff
@@ -16,29 +16,30 @@ code they contain.
     >>> turtle.forward(10)         # actioned by use of '()'s
 
 
-All turtle instructions are examples of calling functions attached 
+All turtle instructions are examples of calling functions attached
 to the turtle object.
 
 `print` is another function::
-    
+
     >>> print('hello')
 
 `print` simply prints its parameter to the console.
 
 .. tip::
 
-    Functions and methods are very similar. 
+    Functions and methods are very similar.
```

Comment:
Do we want to differentiate between methods and functions? Technically, methods are functions that can be bound to an object, but we don't seem to go into it for the rest of the course so maybe we shouldn't make the distinction here....